javascripts/govuk/analytics/tracker.js

```diff
@@ -6,18 +6,25 @@
   // https://github.com/alphagov/govuk_frontend_toolkit/blob/master/docs/analytics.md#create-an-analytics-tracker
 
   var Tracker = function(config) {
-    this.universal = new GOVUK.GoogleAnalyticsUniversalTracker(config.universalId, config.cookieDomain);
-    this.classic = new GOVUK.GoogleAnalyticsClassicTracker(config.classicId, config.cookieDomain);
+    this.trackers = [];
+    if (typeof config.universalId != 'undefined') {
+      this.trackers.push(new GOVUK.GoogleAnalyticsUniversalTracker(config.universalId, config.cookieDomain));
+    }
+    if (typeof config.classicId != 'undefined') {
+      this.trackers.push(new GOVUK.GoogleAnalyticsClassicTracker(config.classicId, config.cookieDomain));
+    }
   };
 
   Tracker.load = function() {
-    GOVUK.GoogleAnalyticsClassicTracker.load();
-    GOVUK.GoogleAnalyticsUniversalTracker.load();
+    for (var i=0; i < this.trackers.length; i++) {
```

Comment:
This method is called statically before a Tracker object is instantiated. So `this.trackers` won't exist at this stage.